src/Event/EventControllerProvider.php

```diff
@@ -37,32 +37,21 @@ function (Application $app) {
         /* @var ControllerCollection $controllers */
         $controllers = $app['controllers_factory'];
 
-        $controllers
-            ->get('event/{cdbid}', 'event_controller:get')
-            ->bind('event');
-        $controllers->delete('event/{cdbid}', 'event_editing_controller:deleteEvent');
-
-        $controllers
-            ->get('event/{cdbid}/history', 'event_controller:history')
-            ->bind('event-history');
-
-        $controllers->post('event', "event_editing_controller:createEvent");
-
-        $controllers->post('event/{itemId}/images', 'event_editing_controller:addImage');
-        $controllers->post('event/{itemId}/images/main', 'event_editing_controller:selectMainImage');
-        $controllers->post('event/{itemId}/images/{mediaObjectId}', 'event_editing_controller:updateImage');
-        $controllers->delete('event/{itemId}/images/{mediaObjectId}', 'event_editing_controller:removeImage');
-
-        $controllers->post('event/{cdbid}/nl/description', 'event_editing_controller:updateDescription');
-        $controllers->post('event/{cdbid}/typical-age-range', 'event_editing_controller:updateTypicalAgeRange');
-        $controllers->delete('event/{cdbid}/typical-age-range', 'event_editing_controller:deleteTypicalAgeRange');
-        $controllers->post('event/{cdbid}/major-info', 'event_editing_controller:updateMajorInfo');
-        $controllers->post('event/{cdbid}/bookingInfo', 'event_editing_controller:updateBookingInfo');
-        $controllers->post('event/{cdbid}/contactPoint', 'event_editing_controller:updateContactPoint');
-        $controllers->post('event/{cdbid}/organizer', 'event_editing_controller:updateOrganizer');
-        $controllers->delete('event/{cdbid}/organizer/{organizerId}', 'event_editing_controller:deleteOrganizer');
-        $controllers->put('event/{cdbid}/audience', 'event_editing_controller:updateAudience');
-        $controllers->post('event/{cdbid}/copies/', 'event_editing_controller:copyEvent');
+        $controllers->post('/', "event_editing_controller:createEvent");
+        $controllers->get('/{cdbid}', 'event_controller:get');
+        $controllers->put('/{cdbid}/audience', 'event_editing_controller:updateAudience');
+        $controllers->put('/{cdbid}/bookingInfo', 'event_editing_controller:updateBookingInfo');
+        $controllers->put('/{cdbid}/contactPoint', 'event_editing_controller:updateContactPoint');
+        $controllers->put('/{cdbid}/majorInfo', 'event_editing_controller:updateMajorInfo');
+        $controllers->put('/{cdbid}/organizer/{organizerId}', 'event_editing_controller:updateOrganizer');
+        $controllers->delete('/{cdbid}/organizer/{organizerId}', 'event_editing_controller:deleteOrganizer');
+        $controllers->put('/{cdbid}/typicalAgeRange', 'event_editing_controller:updateTypicalAgeRange');
+        $controllers->delete('/{cdbid}/typical-age-range', 'event_editing_controller:deleteTypicalAgeRange');
```

Comment:
Deze mag ook nog `typicalAgeRange` worden ipv `typical-age-range`, om consistent te blijven met de property names in de JSON-LD.